akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/AccountExampleWithNullState.java

```diff
@@ -206,7 +201,7 @@ public static AccountEntity create(String accountNumber) {
       return new AccountEntity(accountNumber);
     }
 
-    public AccountEntity(String accountNumber) {
+    private AccountEntity(String accountNumber) {
```

Comment:
Is that right?

Reply:
yes, it's a good convention to make the constructor `private` since the static `create` factory method is supposed to be used.

Reply:
Did not see the statics, thanks.